### PR TITLE
Add yml file

### PR DESCRIPTION
**Pull request by athai**
Created .travis.yml file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: java
+
+jdk:
+  - oraclejdk8
+  - oraclejdk7
```
Comment (leel8on) [inline .travis.yml:7]: Any particular reason for having both 7 and 8? Probably not a bad idea, I just don't know if there is a specific reason to do so.

Comment (athai) [inline .travis.yml:7]: I just thought the extra coverage would be nice.

Comment (leel8on) [inline .travis.yml:7]: Works for me. :+1:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: java
```
Comment (IStallcup) [inline .travis.yml:3]: Why do we use java? Just kidding, looks great other than aforementioned 7 and 8 issue.

**Pull request merged**